src/Http/CookieHelper.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+/**
+ * Copyright 2014 SURFnet bv
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\StepupBundle\Http;
+
+use Symfony\Component\HttpFoundation\Cookie;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Read and write a given cookie from HTTP Requests/Responses.
+ */
+final class CookieHelper
+{
+    /**
+     * @var Cookie
+     */
+    private $cookie;
+
+    public function __construct(Cookie $cookie)
+    {
+        $this->cookie = $cookie;
```

Comment:
Just go with the Cookie :+1: Could you make it explicit in the parameter and field names that this provides the values for the actual cookie that is written? Maybe `cookieSettings`.